L1Trigger/Phase2L1ParticleFlow/python/l1ParticleFlow_cff.py

```diff
@@ -219,6 +252,7 @@
     l1ParticleFlow_pf_barrel +
     l1ParticleFlow_pf_hgcal +
     l1ParticleFlow_pf_hf +
+    l1ParticleFlow_pf_tsa +
```

Comment:
I have taken it out of the sequence and moved it into my own, private configuration file. That way it will not show up in the sequence for the purposes of the PR.